cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteServiceManager.java

```diff
@@ -40,10 +42,10 @@
   private final ServiceClient serviceClient;
   private final Id.Service serviceId;
 
-  public RemoteServiceManager(Id.Service programId, ClientConfig clientConfig, RESTClient restClient,
+  public RemoteServiceManager(ProgramId programId, ClientConfig clientConfig, RESTClient restClient,
```

Comment:
Not sure if this is necessary. The `programId` here is passed to the super class in the constructor `super(programId, remoteApplicationManager);` where `programId` is expected to be `ProgramId`